src/AppBundle/Controller/ProcurationManagerController.php

```diff
@@ -36,6 +36,7 @@ public function indexAction(Request $request): Response
             'totalCount' => $requestsRepository->countManagedBy($this->getUser(), $filters),
             'countToProcess' => $requestsRepository->countToProcessManagedBy($this->getUser()),
             'filters' => $filters,
+            'filterqQueryString' => http_build_query($request->query->all()),
```

Comment:
There is a typo in the view var parameter name... and what about `$request->getQueryString()`?